.github/workflows/check-pr.yml

```
@@ -5,7 +5,7 @@ on:
     branches:
       - develop
       - master
-      - 'v[0-9]+.[0-9]+.[0-9]+'
+      - 'v[0-9]+.[0-9]+.[0-9]+(.*)'
```

```suggestion
      - 'v[0-9]+.[0-9]+.[0-9]+(?:-(?:beta|next)(?:\.\d)?)?'
```
Think this should work

(https://regex101.com/r/Sxuygh/2)